lib/theme_check/corrector.rb

```diff
@@ -9,27 +9,23 @@ def initialize(template:)
     def insert_after(node, content)
       line = @template.full_line(node.line_number)
       line.insert(node.range[1] + 1, content)
-      @template.update!
```

Comment:
This was to prevent re-writing all template files that haven't been modified after correction. To prevent touching files needlessly.

Reply:
I thought I put a check in FileSystemStorage that prevents against this in the write function.

But I forgot :upside_down_smile:

My initial idea was to prevent against this by comparing updated_source with source before writing. Must have lost it in the intermediate refactor.

Reply:
Just changed it here
https://github.com/Shopify/theme-check/pull/148/files#diff-bcabb8a994ed8297a048e60941f0496b0d45c2d209aa04e995276868241d0713R23-R29